docs/tutorials/ruby-driver-collection-tasks.txt

```diff
@@ -101,53 +101,3 @@ documents must contain an ``age`` field which is a number.
                    { '$type' => "number" }
                  }
              })
-
-
-Change Streams
```

Comment:
What do you think of putting a link here to the change stream documentation (instead of removing it) so that existing links people have to this section (using a hashref) will still give them some info instead just going to the top of the page and not having any info about change streams?

Reply:
Okay will do that.